manim/scene/scene_file_writer.py

```diff
@@ -77,46 +77,48 @@ def init_output_directories(self, scene_name):
             )
 
         if file_writer_config["write_to_movie"]:
-            if file_writer_config["video_dir"]:
-                if not file_writer_config["custom_folders"]:
-                    movie_dir = guarantee_existence(
-                        os.path.join(
-                            file_writer_config["video_dir"],
-                            module_directory,
-                            self.get_resolution_directory(),
-                        )
-                    )
-                else:
-                    movie_dir = guarantee_existence(
-                        os.path.join(file_writer_config["video_dir"])
-                    )
-            self.movie_file_path = os.path.join(
-                movie_dir,
-                add_extension_if_not_present(
-                    default_name, file_writer_config["movie_file_extension"]
-                ),
-            )
-            self.gif_file_path = os.path.join(
-                movie_dir,
-                add_extension_if_not_present(default_name, GIF_FILE_EXTENSION),
-            )
+            pass
```

Comment:
Issue: it is not obvious to me what this is doing, or why it is necessary.

Honestly, instead of dealing with this here (file writer logic is *extremely messy*), would it be possible to revert these changes and just keep this PR to adding examples? Or do you need to fix these warnings to add these examples? (If so, why?) FileWriter is not tested nearly as much as it should be, so even if this passes the tests this is almost guaranteed to break some scenes.

Reply:
I split the pull requests